src/themes/greyVest/Tabs.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import TabList from './TabList'
+import Box from './Box'
+
+/* Usage */
+/*
+<Tabs tabControl={ButtonRadio} value="autoLabeled">
+  <Tab value="references" label="References">
+    <PO Table />
+  </Tab>
+  <Tab label="Fancy Label">
+  </Tab>
+  <TabOption value="option2">
```

Comment:
Why is it named `TabOption`. Isn't it just for specifying the label?

Also. Is this for compatibility reasons? It's a weird approach to specify a label.

Reply:
The reason this component exists is to handle scenarios where there's a lot of JSX in the labels (e.g. because they also include result counts, etc). Forcing those into `label` on `Tab` is a lot messier.

We could call this `TabLabel` if that's more clear. It's called `TabOption` because it represents the "option" in the list of tabs (which is the name of the prop on tab list and follows the same API as other select/picker things we have). Here's `TabList`'s `options prop:

https://github.com/smartprocure/contexture-react/blob/master/src/themes/greyVest/TabList.js#L5

Reply:
> The reason this component exists is to handle scenarios where there's a lot of JSX in the labels

Yup that's why there's also `<label for="#id">....</label>`

>  It's called TabOption because it represents the "option" in the list of tabs

That's consistent from an internal point of view. API-wise, `<TabLabel>` is clearer imo, since that's exactly what it is.